app/controllers/bananas2_controller.rb

```diff
@@ -72,7 +72,7 @@ def set_banana
     end
 
     # Never trust parameters from the scary internet, only allow the white list through.
-    def banana_params
+    def banana_params(k:)
```

Comment:
__[rubocop]__
* Lint/UnusedMethodArgument: Unused method argument - `k`. You can also write as `banana_params(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)